### ISSUE TITLE
Unsupported Hive

### ISSUE THREAD

**Issue by chakafasano88** [ironSource/node-regedit#110]
I too am having the same issue. I am receiving the error "Unsupported Hive". The funny thing is that `putValue()` still writes to the registry and stores the value correctly.  It feels like the error is being thrown incorrectly.

```     
const HKCU_KEY = 'HKCU\\SOFTWARE\\MyApp'
await regedit.promisified.putValue({
          HKCU_KEY: {
            PAC: {
              value: PAC,
              type: 'REG_DEFAULT'
            }
          }
        })
      }

**Comment by pash2048** [comment]
I am getting the same error. and reverting to 5.0.0 fixed my issue too, what's the problem with the latest version?

**Comment by liangskyli** [comment]
v5.1.1 the same error

**Comment by demenik** [comment]
duplicate: #11

**Comment by kessler** [comment]
this is fixed now, sorry for the delayed response to this. published on 5.1.2
